confidant/routes/v1.py

```diff
@@ -50,14 +49,24 @@ def get_user_info():
         response = jsonify({'email': None})
     return response
 
+@app.route('/v1/user/role', methods=['GET', 'POST'])
+@authnz.require_auth
+def get_role_info():
```

Comment:
Any reason not to include this info in the `get_user_info()` endpoint?

Reply:
Yeah, I think that would be a better place for this endpoint

Reply:
Looking at the get_user_info() it uses the route '/v1/user/email' so doesn't make sense to put role here. Might be better making an endpoint like '/v1/user/userinfo' and adding email and role details here.

Reply:
I think it would be ok to rename the endpoint to `/v1/user/userinfo`. No reason it needs to be `/v1/user/email`. I'm pretty sure it's just used by the frontend to display the user's email address, so it should be an easy change.